src/docs/forms.njk

```diff
@@ -249,8 +249,18 @@ title: Forms
       <li>Labels always visible</li>
       <li>A new row for each question</li>
       <li>
-        Use a hint to explain any validation rules which have been applied to
-        the input
+        Where there is an external constraint that requires a character limit:
+        <ul>
+          <li>
+            Display how many characters the user has remaining as they type.
+          </li>
+          <li>
+            Display the remaining characters below the text area.
+          </li>
+          <li>
+            Allow the user to enter more characters than the maximum limit. The form validation will handle any excess.
+          </li>
+        </ul>
```

Comment:
I've tried to capture things correctly from [here](https://techfromsage.atlassian.net/wiki/spaces/UT/pages/824082487/Talis+Design+System+Feedback#Communicating-character-limits-and-remaining-character-information), but not 100% sure I have.

Reply:
I think I'd try to include the line "Only use the character count component when there is a good reason for limiting the number of characters users can enter".

So maybe it could be that followed by "when using this component you should:"